js/main.js

```diff
@@ -0,0 +1,314 @@
+const buttons = document.getElementsByClassName("button");
+const digitButtons = document.getElementsByClassName("digitButton");
+const screen = document.getElementById("screen");
+const unblockedKeys = [8, 13, 16, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 114, 187];
+
+let calcType = null;
+let result = null;
+let factor2= null;
+let newNumber = true;
+let dropTheResult = false;
+
+const calculation = function () {
+    switch (calcType) {
+        case 1:
```

Comment:
In a current variant not clear which type of math operation we are going to do. Use string representation of math operation like 'division' instead of numbers.